_includes/configuration/text-color.md

```diff
@@ -34,9 +18,12 @@ By default, the number of rows and columns is dependent of the number of colors
 
 ```js
 tinymce.init({
-  selector: "textarea",  // change this value according to your HTML
-  toolbar: "forecolor backcolor",
-  color_cols: "5"
+  selector: 'textarea',  // change this value according to your HTML
+  menu: {
```

Comment:
This isn't needed and is more likely to cause confusion, as these are in the format menu by default.

Reply:
Removed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/ad7a0e6d92f1cefdf4676efc4d09e53a8073cad3